libs/domain/picking/src/components/picking/picking.component.ts

```diff
@@ -56,29 +145,117 @@ export class PickingComponent extends PickingListMixin(LitElement) {
       : html``;
   }
 
-  protected renderTabContents(): TemplateResult {
-    return this.tabs
-      ? html`${repeat(
-          this.tabs,
-          (tab) => html`<div slot="panels" id="tab-${tab.id}">
-            ${tab.items?.length
-              ? repeat(
-                  tab.items,
-                  (item) =>
-                    html`<oryx-picking-product-card
-                      .productItem=${item.product}
-                    >
-                      ${item.product.sku} ${item.product.productName}
-                    </oryx-picking-product-card>`
-                )
-              : this.renderFallback()}
-          </div>`
-        )}`
-      : html``;
+  protected renderTabContents(tabs: PickingTab[]): TemplateResult {
+    return html`
+      ${when(
+        tabs.length,
+        () =>
+          html`${repeat(
+            tabs,
+            (tab) => html`<div
+              slot="panels"
+              id="tab-${tab.id}"
+              class="tab-panels"
+            >
+              ${when(
+                tab.items?.length,
+                () => html`
+                  ${repeat(
+                    tab.items,
+                    (item) => item.product.id,
+                    (item) =>
+                      html`
+                        <oryx-picking-product-card
+                          .productItem=${item}
+                          .status=${tab.id}
+                          @oryx.submit=${this.savePickingItem}
+                          @oryx.edit=${this.editPickingItem}
+                        ></oryx-picking-product-card>
+                      `
+                  )}
+                `,
+                () => this.renderFallback()
+              )}
+            </div>`
+          )}`
+      )}
+    `;
   }
 
   protected renderFallback(): TemplateResult {
-    return html`<p>${i18n('picking.no-picking-items-found')}</p>`;
+    const pickedItems = this.pickingList?.items.filter(
+      (item) => item.status === ItemsFilters.Picked
+    );
+
+    const isPickingComplete =
+      pickedItems?.length === this.pickingList?.items.length;
+
+    return html`
+      ${when(
+        isPickingComplete,
+        () =>
+          html`
+            <div class="picking-complete">
+              <div class="img-wrap">
+                <oryx-image resource="picking-items-processed"></oryx-image>
+              </div>
+              <h3 class="title-empty">${i18n(`picking.great-job`)}!</h3>
+              <p>${i18n(`picking.all-items-are-processed`)}!</p>
+            </div>
+          `
+      )}
+    `;
+  }
+
+  protected renderConfirmationModal(): TemplateResult {
+    return html`
+      <oryx-modal
+        ?open=${this.isConfirmPickingDialogOpen}
+        enableFooter
+        preventclosebybackdrop
+      >
+        <div slot="heading">
+          ${i18n('picking.product-card.confirm-picking')}
+        </div>
+
+        <span>
+          You only picked
```

Comment:
Add this ToDo please:
`//ToDo: should be refactored when i18n is supporting elements as a parts of templates`

Reply:
let's just use the `${i18n('picking.product-card.you-only-picked')}` for now